provisioning-telephony/config.yml

```diff
@@ -0,0 +1,72 @@
+---
+title: Provisioning Telephony
+description: This tutorial implements BYOC Cloud solution with a cloud-based carrier 
+categories:
+- Trunk
+- Telephony
+steps:
+- title: Introduction
+  content: |
+    This tutorial walks through the steps on how to implement BYOC Cloud in Genesys Cloud using [Purecloud Platform API](https://developer.mypurecloud.com/api/rest/client-libraries/index.html). In order for BYOC Cloud to work with a third-party carrier, the carrier must:
+    
+    * Be SIP-compliant
+    * Support the UDP or TCP trunk transport protocol
+    * Have a publicly routable IP address or host name
+      * Carriers must fully support Record-Routing as required by RFC 3261 to ensure proper routing of sequential SIP requests.
+    
+    For more information about carrier requirements, check Genesys Resource Center article [Carrier requirements for BYOC Cloud](https://help.mypurecloud.com/articles/carrier-requirements-byoc-cloud/).
+    
+    To successfully run the code, a JSON file is needed to provide the necessary input. In this tutorial, the JSON file is titled input-template.json.
+
+- title: JSON Files and Variables
+  content: |
+    This file is used as substitute to input text box. This is where the user enters all the necessary details. Key value pair has its corresponding description. Please make sure to replace all the value of keys from description to actual data and fill all the required fields to proceed. 
+    
+    Please refer to [BYOC Cloud public SIP IP addresses](https://help.mypurecloud.com/articles/byoc-cloud-public-sip-ip-addresses/) for instance or Public SIP Addresses.  
+
+- title: Authentication
+  content: |
+    Genesys Cloud will verify user's credentials using Client Credential authentication. Once the validation is successful, it will proceed with the succeeding process.
+
+- title: Check BYOC Feature
+  content: |
+    Get the list of products available for the org used by calling [getAuthorizationProducts()](https://developer.mypurecloud.com/api/rest/client-libraries/javascript/AuthorizationApi.html#organizationproductentitylisting_getauthorizationproducts__) function of Authorization API and check if the org has BYOC capability. If the org is BYOC capable, it will proceed to create location function.  Please take note that only BYOC enabled orgs can proceed.
```

Comment:
How would a customer get BYOC enabled for their org? It would be good to state "Customers can enable BYOC for their org by..." and link to a resource center article or state whatever they need to do.

Reply:
As far as I know. If the partner/customer wants to have their org BYOC enabled. They need to contact Zach Hinkle and he will take care of it. Let me verify this first with Jim.

Reply:
If that's the case, we probably want to state to open a case with Care. I'd prefer not to document instructions to contact a specific person.

Reply:
It is a license they have to purchase and onboarding will enable the feature. It might be good to suggest contacting their sales representative or account manager.